python/ray/scripts/scripts.py

```diff
@@ -233,14 +238,35 @@ def dashboard(cluster_config_file, cluster_name, port):
     "--include-webui",
     default=None,
     type=bool,
-    help="provide this argument if the UI should be started")
+    help="provide this argument if the UI should be started "
+    "(DEPRECATED: please use --include-dashboard")
 @click.option(
     "--webui-host",
     required=False,
     default="localhost",
-    help="The host to bind the web UI server to. Can either be localhost "
-    "(127.0.0.1) or 0.0.0.0 (available from all interfaces). By default, this "
-    "is set to localhost to prevent access from external machines.")
+    help="the host to bind the dashboard server to, either localhost "
+    "(127.0.0.1) or 0.0.0.0 (available from all interfaces). By default,"
+    " this is localhost."
+    " (DEPRECATED: please use --dashboard-host)")
+@click.option(
+    "--include-dashboard",
+    default=None,
```

Comment:
Should we be starting it by default? A question for another PR

Reply:
I am confused. It must be started by default right? Why is i the question for another PR?

Reply:
We do start it by default already. In retrospect, my comment description here could've been more clear. I will rewrite. The default arg of None starts the dashboard.

Reply:
Oh, interesting. So you have to set `--include-dashboard=False` to not start it? I think I was confused by the flag name.